workflows/bacterial_base.nf

```diff
@@ -90,13 +91,16 @@ workflow CALL_BACTERIAL_BASE {
 
         post_align_qc(bwa_mem_ref.out.bam, referenceGenome, coreLociBed)
 
+        nanoplot(ch_input_meta)
```

Comment:
```suggestion
        nanoplot(ch_reads_w_meta)
```

Reply:
Check the modules that use `ch_reads_w_meta` and make sure the input channels are the same for `nanoplot`. Should work after that!